pkg/vdri/trustbloc/vdri.go

```diff
@@ -349,8 +348,8 @@ func WithAuthToken(authToken string) Option {
 }
 
 // EnableSignatureVerification enables signature verification
-func EnableSignatureVerification() Option {
+func EnableSignatureVerification(enable bool) Option {
```

Comment:
you should add test for this - it will fix your coverage too

Reply:
Done now